### PR TITLE
Add achievement secrets

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

File: achievements/secrets.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Symfony Secrets
+
+The EU-FOSSA hackathon was the opportunity to discussed and define a simple
```
Comment (OskarStark) [inline achievements/secrets.md]: s/discussed/discuss/

File: achievements/secrets.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+and flexible solution to use sensitive informations (like passwords,
+authentication keys, ...) in a symfony application.
```
Comment (OskarStark) [inline achievements/secrets.md]: s/symfony/Symfony/

File: achievements/secrets.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+It solves the following common issue:
+
+> How to define new sensitive parameter without exposing theme in a
```
Comment (OskarStark) [inline achievements/secrets.md]: s/theme/them/

File: achievements/secrets.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  Repository, nor configuring manually a server each time the projet requires
```
Comment (OskarStark) [inline achievements/secrets.md]: s/Repository/repository/

File: achievements/secrets.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  a new ENV variable
```
Comment (OskarStark) [inline achievements/secrets.md]: s/ENV/env/

File: achievements/secrets.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+Thank to @mpdude, @tobion, @nicolas-grekas and @jderusse, we defined the
```
Comment (OskarStark) [inline achievements/secrets.md]: s/Thank/Thanks/

File: achievements/secrets.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+use-cases that would be covered by that new feature, we desin the main
```
Comment (OskarStark) [inline achievements/secrets.md]: s/desin/design/

**Review by OskarStark — CHANGES_REQUESTED** [review]
Nice article 👍🏻

File: achievements/secrets.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+and flexible solution to use sensitive informations (like passwords,
+authentication keys, ...) in a Symfony application.
+
+It solves the following common issue:
+
+> How to define new sensitive parameter without exposing them in a
+  repository, nor configuring manually a server each time the projet requires
+  a new e,v variable
+
+Thanks to @mpdude, @tobion, @nicolas-grekas and @jderusse, we defined the
```
Comment (jfhovinne) [inline achievements/secrets.md]: Could you link to the accounts ?

File: achievements/secrets.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+use-cases that would be covered by that new feature, we design the main
```
Comment (jfhovinne) [inline achievements/secrets.md]: 'use cases' and 'designed'

File: achievements/secrets.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+interfaces and ended with a working Proof of Concept.
+
+[Click here for the RFC issue](https://github.com/symfony/symfony/issues/27351).
```
Comment (jfhovinne) [inline achievements/secrets.md]: Please avoid 'Click here' - https://www.w3.org/QA/Tips/noClickHere

File: achievements/secrets.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+
+The EU-FOSSA hackathon was the opportunity to discuss and define a simple
+and flexible solution to use sensitive informations (like passwords,
+authentication keys, ...) in a Symfony application.
+
+It solves the following common issue:
+
+> How to define new sensitive parameter without exposing them in a
+  repository, nor configuring manually a server each time the projet requires
+  a new e,v variable
```
Comment (jfhovinne) [inline achievements/secrets.md]: e,v

**Review by jfhovinne — CHANGES_REQUESTED** [review]
Thanks! Minor fixes.

File: achievements/secrets.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+> How to define new sensitive parameters without exposing them in a
+  repository, nor configuring manually a server each time the projet requires
+  a new env variable
+
+Thanks to [@mpdude](https://github.com/mpdude), [@tobion](https://github.com/tobion),
+[@nicolas-grekas](https://github.com/nicolas-grekas) and [@jderusse](https://github.com/jderusse),
+we defined the use cases that would be covered by that new feature, we designed the main
+interfaces and ended with a working Proof of Concept.
+
+This subject was in initialized in [the RFC issue](https://github.com/symfony/symfony/issues/27351).
```
Comment (jderusse) [inline achievements/secrets.md]: Shall we add a link to https://www.webfactory.de/blog/storing-secrets-for-symfony-applications ? ping @mpdude

Comment (mpdude) [reply achievements/secrets.md]: Yes sure, why not?

Comment (jderusse) [reply achievements/secrets.md]: Added.

> why not?

politely asking it :-)

File: achievements/secrets.md (hunk 8 above)
Comment (jderusse) [reply achievements/secrets.md]: done. Thanks for the review

File: achievements/secrets.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+> How to define new sensitive parameters without exposing them in a
+  repository, nor configuring manually a server each time the projet requires
+  a new env variable
+
+Thanks to [@mpdude](https://github.com/mpdude), [@tobion](https://github.com/tobion),
+[@nicolas-grekas](https://github.com/nicolas-grekas) and [@jderusse](https://github.com/jderusse),
+we defined the use cases that would be covered by that new feature, we designed the main
+interfaces and ended with a working Proof of Concept.
+
+This subject was in initialized in [the RFC issue](https://github.com/symfony/symfony/issues/27351),
```
Comment (jfhovinne) [inline achievements/secrets.md:18]: "was initialised"

File: achievements/secrets.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+during the hackathon, [@mpdude](https://github.com/mpdude) wrote
+[a nice blog post](https://www.webfactory.de/blog/storing-secrets-for-symfony-applications)
+diging into that topic.
```
Comment (jfhovinne) [inline achievements/secrets.md:21]: digging

**Review by jfhovinne — CHANGES_REQUESTED** [review]
Typos.

**Comment by jfhovinne** [comment]
Thanks. Will fix typos.

**Pull request merged**